modules/Sources/Features/AddressBook/AddressBookStore.swift

```diff
@@ -266,21 +266,21 @@ public struct AddressBook {
                     return .none
                 }
 
-            case let .fetchedABContacts(abContacts, _):
+            case let .fetchedABContacts(abContacts, requestToSync):
                 state.addressBookContacts = abContacts
-//                if requestToSync {
-//                    return .run { send in
-//                        do {
-//                            let syncedContacts = try await addressBook.syncContacts(abContacts)
-//                            await send(.fetchedABContacts(syncedContacts, false))
-//                        } catch {
-//                            print("__LD syncContacts Error: \(error.localizedDescription)")
-//                            // TODO: FIXME
-//                        }
-//                    }
-//                } else {
+                if requestToSync {
+                    return .run { send in
+                        do {
+                            let syncedContacts = try await addressBook.syncContacts(abContacts)
+                            await send(.fetchedABContacts(syncedContacts, false))
+                        } catch {
+                            print("__LD syncContacts Error: \(error.localizedDescription)")
+                            // TODO: FIXME
```

Comment:
I know this is an existing TODO, but please file an issue and link it.

Reply:
Follow up #1408